src/components/datepicker/js/datepickerDirective.js

```diff
@@ -53,7 +53,7 @@
    * </hljs>
    *
    */
-  function datePickerDirective($$mdSvgRegistry) {
+  function datePickerDirective($$mdSvgRegistry, $mdUtil) {
```

Comment:
Isn't that the same as in #9049. You could probably drop #9049 and just use this one.